index.js

```diff
@@ -0,0 +1,77 @@
+/* eslint-disable no-use-before-define */
+/* eslint-disable max-classes-per-file */
+import { DateTime } from './modules/luxon.js';
+import StoreBookData from './modules/storage.js';
+import Book from './modules/book.js';
+
+const liList = document.getElementById('li-list');
+const liAdd = document.getElementById('li-add');
+const liContact = document.getElementById('li-contact');
+const list = document.getElementById('list');
+const addNew = document.getElementById('addNew');
+const contact = document.getElementById('contact');
+
+liList.addEventListener('click', (e) => {
+  e.preventDefault();
+  list.style.display = 'block';
+  addNew.style.display = 'none';
+  contact.style.display = 'none';
+});
+
+liAdd.addEventListener('click', (e) => {
+  e.preventDefault();
+  addNew.style.display = 'block';
+  list.style.display = 'none';
+  contact.style.display = 'none';
+});
+
+liContact.addEventListener('click', (e) => {
+  e.preventDefault();
+  contact.style.display = 'flex';
+  addNew.style.display = 'none';
+  list.style.display = 'none';
+});
+
+const dateElement = document.getElementById('date');
+
+const date = DateTime.now();
+dateElement.textContent = date.toLocaleString(DateTime.DATETIME_MED);
+
+class Layout {
+  static displayBooks() {
+    const storedBooks = StoreBookData.getBooksData();
+    storedBooks.map((book) => Layout.bookList(book));
+  }
+
+  static bookList(book) {
+    const collection = document.getElementById('collection');
+    collection.innerHTML += `
+      <li id='${book.id}' class='listItem'>
+        <p>"${book.title}" by ${book.author} </p>
+        <button id='delete' type='button' onclick ='Layout.deleteBook(${book.id})'>Remove</button>
+      </li>
```

Comment:
- [ ] When I click on the Remove button it's expected to delete a book from the list of books. Currently, it does not seem to be working as expected. Adding books is working fine but deleting a book is not working. Kindly fix that feature so your project will be complete for the milestone 👍

Reply:
Please ignore the review headline above. I meant to request changes and not approve. Please work on the changes as highlighted above.

![image](https://user-images.githubusercontent.com/50941074/207268785-87f189b3-3791-42a4-be46-8e239af65a22.png)